recipes/thriftpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: thriftpy
+  version: "0.3.7"
+
+source:
+  fn: thriftpy-0.3.7.tar.gz
+  url: https://pypi.python.org/packages/source/t/thriftpy/thriftpy-0.3.7.tar.gz
+  md5: f7bda2f5fe4f6eee0e3fa0949ea231ef
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ply >=3.4,<4.0
+    - cython
```

Comment:
Can we please also add a `# [not win]` selector to `cython` here? Needs to be 2 spaces from the package.